### PR TITLE
ignore deepsource analysis on .pb.go and example code

### PR DESCRIPTION
**Pull request by kevindiu**
### Description:

This PR disable deepsource analysis on generated file `.pb.go` and example code.
Ref: https://deepsource.io/docs/config/deepsource-toml.html#exclude-patterns

### Environment:

- Go Version: 1.15.2
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.1

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [x] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .deepsource.toml [hunk 1]
```diff
@@ -15,6 +15,10 @@
 #
 version = 1
 
+exclude_patterns = [
+  "**/*.pb.go"
```
Comment (kpango) [inline .deepsource.toml:19]: could you please exclude example dir & _test.go & _mock_test.go & bench_test.go file ?

Comment (kevindiu) [reply .deepsource.toml:19]: I think I need some discussion in team about excluding `_test.go` files as I think it may affect our testing code quality.

Comment (kpango) [reply .deepsource.toml:19]: I see. okay, please exclude example dir

**Comment by vankichi** [comment]
I think example code should be excluded from DeepSource.
These codes are not influences any other Vald codes.

```go
// if you want to check `err` please define var of error
res, _ := any.Method()
```

**Review by kpango — APPROVED** [review]
LGTM

**Review by hlts2 — APPROVED** [review]
LGTM

**Pull request merged**